rsocket-core/src/main/java/io/rsocket/fragmentation/FragmentationDuplexConnection.java

```diff
@@ -53,16 +55,14 @@
    * @param mtu the fragment size, greater than {@link #MIN_MTU_SIZE}
    * @param type a label to use for logging purposes
    */
-  public FragmentationDuplexConnection(DuplexConnection delegate, int mtu, String type) {
-    super(delegate);
+  public FragmentationDuplexConnection(
+      DuplexConnection delegate, int mtu, int maxReassemblySize, String type) {
+    super(delegate, maxReassemblySize);
```

Comment:
Maybe we can call this `maxPayloadSize` to align the terminology, and also in the base class?

Reply:
agree